rllib/utils/actor_manager.py

```diff
@@ -696,10 +700,24 @@ def fetch_ready_async_reqs(
             The values may be actual data returned or exceptions raised during the
             remote call in the format of RemoteCallResults.
         """
+        if isinstance(tags, str):
+            tags = {tags}
+        elif isinstance(tags, list):
+            tags = set(tags)
+        else:
+            raise ValueError(
+                f"tags must be either a str or a list of str, got {type(tags)}."
+            )
+        remote_calls = []
+        remote_actor_ids = []
```

Comment:
actor_ids probably needs to be a set.

Reply:
hmm actually I'm not sure if this is the case. Remote actor ids directly corresponds to remote_calls and needs to be the same length. There is one remote call that corresponds to a actor id.